### ISSUE TITLE
Blank screen on default snippets overview

### ISSUE THREAD

**Issue by alexandersch** [sulu/sulu#6041]
| Q | A
| --- | ---
| Sulu Version | 2.3.0-RC1
| PHP Version | 7.4.x
| DB Version | MySQL 5.7.x
| Browser Version | Chrome, latest version

# Actual Behavior
Blank screen when going to default snippets overview. Console error logged:
```
react-dom.production.min.js:216 TypeError: Cannot read property 'webspaceKey' of undefined
     at $le.getNode (CacheClearToolbarAction.js:31)
     at i.render (SnippetAreas.js:160)
     at Te (mobx.module.js:1015)
     at index.module.js:847
     at me (mobx.module.js:801)
     at e.track (mobx.module.js:1833)
     at i.n [as render] (index.module.js:841)
     at Hs (react-dom.production.min.js:187)
     at Fs (react-dom.production.min.js:186)
     at Ha (react-dom.production.min.js:269)
```

# Expected Behavior
Seeing the default snippets overview

# Steps to Reproduce
Go to pages -> default snippets

**Comment by niklasnatter** [comment]
Hey,
thanks for reporting! This should be fixed in #6042.

To install a version that includes the fix in your project, you can adjust the version constraint for the `sulu/sulu` package from `~2.3.0-RC1` to `^2.3@dev` in your `composer.json` and execute `composer update`.
Make sure to also update the Javascript build of the administration interface in you project with the `bin/console sulu:admin:update-build` command afterwards.

**Comment by alexandersch** [comment]
Thank you!
